modules/oe_webtools_maps/oe_webtools_maps.permissions.yml

```diff
@@ -0,0 +1,2 @@
+administer webtools maps form:
```

Comment:
This is not in line with the remaining permissions, it should be `administer webtools maps`
https://github.com/openeuropa/oe_webtools/blob/master/modules/oe_webtools_cookie_consent/oe_webtools_cookie_consent.permissions.yml#L1
https://github.com/openeuropa/oe_webtools/blob/master/modules/oe_webtools_globan/oe_webtools_globan.permissions.yml#L1
https://github.com/openeuropa/oe_webtools/blob/master/modules/oe_webtools_cookie_consent/oe_webtools_cookie_consent.permissions.yml#L1